cookbook/workflow/homestead.rst

```diff
@@ -0,0 +1,66 @@
+.. index:: Vagrant, Homestead
+
+Using Symfony with Homestead/Vagrant
+====================================
+
+In order to develop a Symfony application, you might want to use a virtual
+development environment instead of the built-in server or WAMP/LAMP. Homestead_
+is an easy-to-use Vagrant_ box to get a virtual environment up and running
+quickly.
+
+Install Vagrant and Homestead
+-----------------------------
+
+Before you can use Homestead, you need to install and configure Vagrant and
+Homestead as explained in `the Homestead documentation`_.
+
+Setting Up a Symfony Application
+--------------------------------
+
+Imagine you've installed your Symfony application in
+``~/projects/symfony_demo`` on your local system. You first need Homestead to
+sync your files in this project. Execute ``homestead edit`` to edit the
```

Comment:
Homestead is syncing the local directory with the directory you mapped it to in the Homestead environment, I'll expand the description a bit.